templates/rhdm71-prod-immutable-kieserver.yaml

```diff
@@ -0,0 +1,494 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    iconClass: icon-jboss
+    tags: rhdm,jboss,kieserver,immutable
+    version: "1.0"
+    openshift.io/display-name: Red Hat Decision Manager 7.1 immutable production environment
+    openshift.io/provider-display-name: Red Hat, Inc.
+    description: Application template for an immultable KIE server in a production environment, for Red Hat Decision Manager 7.1
+    template.openshift.io/long-description: This template defines resources needed for a immultable production runtime environment for Red Hat Decision Manager 7.1, including application deployment configuration, secure and insecure http communication and persistent volume to store data that needs to survive to a restart.
```

Comment:
Shouldn't we keep the keyword s2i in some of the descriptions so user can know that this template will perform a s2i build?

Reply:
Done.